variants/models.py

```diff
@@ -3035,6 +3035,27 @@ def __getitem__(self, key):
             return self.__wrapped__.__getitem__(key)
 
 
+class AcmgCriteriaSchema(models.Model):
```

Comment:
Do we want a flag for `factory_settings`?

Reply:
Ahh, I see. Yes, sounds like a good idea.